src/compressed_tensors/utils/safetensors_load.py

```diff
@@ -32,8 +32,12 @@
     "get_nested_weight_mappings",
     "get_quantization_state_dict",
     "is_quantization_param",
+    "get_nested_mappings_from_state_dict",
 ]
 
+WEIGHT_MAPPING_TYPE = Dict[str, str]
+NESTED_WEIGHT_MAPPING_TYPE = Dict[str, WEIGHT_MAPPING_TYPE]
```

Comment:
Nit: Type names should be camel case
https://docs.python.org/3/library/typing.html#type-aliases